docs/actnow.md

```diff
@@ -1,5 +1,5 @@
 # Static Actions
-While Shesmu is mostly about the olives, it _does_ runs actions and it can be
+While Shesmu is mostly about the olives, it _does_ run actions and it can be
```

Comment:
might be nitpicky, but maybe "While Shesmu is mostly about olives" vs "While Shesmu is mostly about the olives"